src/App.tsx

```diff
@@ -1,11 +1,16 @@
 import React from "react";
-import Landing from "@landing";
+import { Routes, Route } from "react-router-dom";
+import { Landing } from "./pages";
+import { SignIn, About } from "@components/index";
```

Comment:
```suggestion
import { SignIn, About } from "@components/index";
import { Landing } from "./pages";
```
Maybe relative paths at the bottom?

Reply:
Sure! I wasn't aware they had an order. Fixed!